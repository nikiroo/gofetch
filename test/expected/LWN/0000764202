                        FIREFOX 62.0 RELEASED                      

  [Development] Sep 5, 2018 17:31 UTC (Wed) (ris)

  o News link: https://lwn.net/Articles/764202/
  o Source link: 


    Mozilla  has  released Firefox 62.0, with several new features.
    The  Firefox  Home (default New Tab) allows users to display up
    to  4  rows  of  top sites, Pocket stories, and highlights; for
    those  using containers there is menu option to reopen a tab in
    a  different  container;  Firefox  63 will remove all trust for
    Symantec-issued  certificates,  and  it  is optional in Firefox
    62;  FreeBSD  support for WebAuthn was added; and more. See the
    [1]release notes for details.
    
    
    
    [1] https://www.mozilla.org/en-US/firefox/62.0/releasenotes/



