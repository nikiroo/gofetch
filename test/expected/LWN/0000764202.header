0Firefox 62.0 released	null/LWN/0000764202		70
i  [Development] Sep 5, 2018 17:31 UTC (Wed) (ris)
i
i    Mozilla  has  released Firefox 62.0, with several new features.
i    The  Firefox  Home (default New Tab) allows users to display up
i    to  4  rows  of  top sites, Pocket stories, and highlights; for
i    those  using containers there is menu option to reopen a tab in
i    a  different  container;  Firefox  63 will remove all trust for
i    Symantec-issued  certificates,  and  it  is optional in Firefox
i    62;  FreeBSD  support for WebAuthn was added; and more. See the
i    [1]release notes for details.
i    
i    
i    
i    [1] https://www.mozilla.org/en-US/firefox/62.0/releasenotes/
i
