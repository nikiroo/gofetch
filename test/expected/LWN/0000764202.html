<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story'>
	<h2>Firefox 62.0 released</h2>
	<div class='details'>([Development] Sep 5, 2018 17:31 UTC (Wed) (ris))</div>
	<br/>
    <ul>
    <li>Reference: <a href=''>0000764202</a></li>
        <li>News link: <a href='https://lwn.net/Articles/764202/'>https://lwn.net/Articles/764202/</a></li>
        <li>Source link: <a href=''></a></li>
    </ul>
	<br/>
	<div class='content' style='text-align: justify'>
		Mozilla has released Firefox 62.0, with several new features. The Firefox Home (default New Tab) allows users to display up to 4 rows of top sites, Pocket stories, and highlights; for those using containers there is menu option to reopen a tab in a different container; Firefox 63 will remove all trust for Symantec-issued certificates, and it is optional in Firefox 62; FreeBSD support for WebAuthn was added; and more. See the [1]release notes for details.<br/><br/><br/><br/>[1] https://www.mozilla.org/en-US/firefox/62.0/releasenotes/
	</div>
<hr/>
</div>
</body>
